std/assembly/util/string.ts

```diff
@@ -652,39 +652,48 @@ export function toUpper8(c: u32): u32 {
 export function strtol<T>(str: string, radix: i32 = 0): T {
   var len = str.length;
   if (!len) {
-    // @ts-ignore: cast
-    if (isFloat<T>()) return <T>NaN;
-    // @ts-ignore: cast
-    return <T>0;
+    if (isFloat<T>()) {
+      // @ts-ignore: cast
+      return <T>NaN;
+    } else {
+      // @ts-ignore: cast
+      return <T>0;
+    }
   }
 
   var ptr = changetype<usize>(str) /* + HEAD -> offset */;
   var code = <u32>load<u16>(ptr);
 
   // determine sign
   // @ts-ignore: cast
-  var sign: T = 1;
+  var sign = false;
```

Comment:
Does this fix something? If so, then there are a couple unnecessary `ts-ignore`s now besides the branch it adds.